modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -193,7 +198,7 @@ def get_filezilla_creds(paths)
     creds.each do |cred|
       credentials << [cred['host'], cred['port'], cred['user'], cred['password'], cred['ssl']]
 
-      session.db_record ? (source_id = session.db_record.id) : (source_id = nil)
+      session.db_record ? session.db_record.id : nil
```

Comment:
This looks like dead code

(Unless it's got some sort of weird side effect that isn't obvious)

Reply:
Removed